### ISSUE TITLE
Expose AJP port

### ISSUE THREAD

**Issue by tokland** [EyeSeeTea/d2-docker#47]
- [ ] d2-docker start --port=8080 --ajp-port=8020

**Comment by tokland** [comment]
@ifoche There is also proxyName, and some others. I think it would be better to add a single option `--tomcat-server-path="my-server.xml"` so the user can customize all they need, otherwise we'll end up with lots of options and less control over tomcat.

**Comment by ifoche** [comment]
@tokland I think you're right, using my-server.xml would provide the user with the capability to change everything, but the user would need to understand all parameters. Given that's for advanced users it might be something acceptable, but then I think we would have to provide them easily with an example to work with, instead of writing it from zero. I don't know how to do that (like generating the default one we're using and maybe in the --help and README explain how to customize?), and I don't know if the user would need to understand the internals of d2-docker for the server.xml to make sense for him/her...what do you think?

**Comment by tokland** [comment]
Yup, it makes to add a section in the readme. The idea to explain is to copy the server.xml from the project as a template and customize, making clear which things cannot be changed (i.e. the 8080 port).
